documentation/provider/sakuracloud.md

```diff
@@ -0,0 +1,99 @@
+This is the provider for [Sakura Cloud](https://cloud.sakura.ad.jp/).
+
+## Configuration
+To use this provider, add an entry to `creds.json` with `TYPE` set to `SAKURACLOUD`
+along with API credentials.
+
+Example:
+
+{% code title="creds.json" %}
+```json
+{
+  "sakuracloud": {
+    "TYPE": "SAKURACLOUD",
+    "access_token": "your-access-token",
+    "access_token_secret": "your-access-token-secret"
+  }
+}
+```
+{% endcode %}
+
+The `endpoint` is optional. If omitted, the default endpoint is assumed.
+
+Endpoints are as follows:
+
+* `https://secure.sakura.ad.jp/cloud/zone/is1a/api/cloud/1.1` (Ishikari first Zone)
+* `https://secure.sakura.ad.jp/cloud/zone/is1b/api/cloud/1.1` (Ishikari second Zone)
+* `https://secure.sakura.ad.jp/cloud/zone/tk1a/api/cloud/1.1` (Tokyo first Zone)
+* `https://secure.sakura.ad.jp/cloud/zone/tk1b/api/cloud/1.1` (Tokyo second Zone)
+
+DNS service is independent of zones, so you can use any of these endpoints.
+The default is the Ishikari first Zone.
+
+Alternatively you can also use environment variables.
+
+```shell
+export SAKURACLOUD_ACCESS_TOKEN="your-access-token"
+export SAKURACLOUD_ACCESS_TOKEN_SECRET="your-access-token-secret"
+```
+
+{% code title="creds.json" %}
+```json
+{
+  "sakuracloud": {
+    "TYPE": "SAKURACLOUD",
+    "access_token": "$SAKURACLOUD_ACCESS_TOKEN",
+    "access_token_secret": "$SAKURACLOUD_ACCESS_TOKEN_SECRET"
+  }
+}
+```
+{% endcode %}
+
+## Metadata
+This provider does not recognize any special metadata fields unique to
+Sakura Cloud.
+
+## Usage
+An example configuration:
+
+{% code title="dnsconfig.js" %}
+```javascript
+var REG_NONE = NewRegistrar("none");
+var DSP_SAKURACLOUD = NewDnsProvider("sakuracloud");
+
+D("example.com", REG_NONE, DnsProvider(DSP_SAKURACLOUD),
+  DefaultTTL(3600),
+  A("test", "192.0.2.1"),
+END);
+```
+{% endcode %}
+
+The default TTL of Sakura Cloud is 3600, so it's recommended to set
+`DefaultTTL(3600)`.
```

Comment:
**Question**: What happens within Sakura Cloud/DNSControl if you do not include this defaultTTL?

Reply:
The DefaultTTL value set by the user is simply set to Sakura Cloud.
I realise that `DefaultTTL(3600)` is a meaningless setting for a user, so I am removing this description.

[SAKURACLOUD: Deletion of description about default TTL (3600)](https://github.com/StackExchange/dnscontrol/pull/3086/commits/c5f7c02916c681364950bf2eac2b713566f92f19)

Reply:
Thanks for the explanation and the code adjustment.